projects/packages/stats-admin/src/class-notices.php

```diff
@@ -70,23 +71,33 @@ public function get_notices_to_show() {
 		$stats_views              = intval( Stats_Options::get_option( 'views' ) );
 		$odyssey_stats_changed_at = intval( Stats_Options::get_option( 'odyssey_stats_changed_at' ) );
 
+		// Check if Jetpack is integrated with the Complianz plugin, which blocks the Stats.
+		$complianz_options_integrations  = get_option( 'complianz_options_integrations' );
+		$is_jetpack_blocked_by_complianz = ! isset( $complianz_options_integrations['jetpack'] ) || $complianz_options_integrations['jetpack'];
+		$is_complianz_installed          = $notices_wpcom[ self::GDPR_COOKIE_CONSENT_NOTICE_ID ];
```

Comment:
ah.. I guess we could remote this line now right?

Reply:
addressed in adc247a977

Reply:
I updated the local installation check to work compatibly with API without the key `GDPR_COOKIE_CONSENT_NOTICE_ID` in [3ea6d10](https://github.com/Automattic/jetpack/pull/37870/commits/3ea6d10c2d5b8741f7a03560e1ff2e72d9303a33).